dev/com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/Packaging_Web.java

```diff
@@ -0,0 +1,249 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jpa.tests.spec10.packaging.tests;
+
+import java.util.HashSet;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import org.jboss.shrinkwrap.api.ArchivePath;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.internal.AssumptionViolatedException;
+import org.junit.runner.Description;
+import org.junit.runner.RunWith;
+import org.junit.runners.model.Statement;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.config.Application;
+import com.ibm.websphere.simplicity.config.ServerConfiguration;
+import com.ibm.ws.jpa.fvt.packaging.xmlmappingfile.annotationoverride.tests.web.TestAnnotationOverrideServlet;
+import com.ibm.ws.jpa.fvt.packaging.xmlmappingfile.mappingfiledefaults.tests.web.TestMappingFileDefaultsServlet;
+import com.ibm.ws.jpa.fvt.packaging.xmlmappingfile.metadatacomplete.tests.web.TestMetadataCompleteServlet;
+import com.ibm.ws.jpa.fvt.packaging.xmlmappingfile.xmlmetadatacomplete.tests.web.TestXMLMetadataCompleteServlet;
+
+import componenttest.annotation.Server;
+import componenttest.annotation.TestServlet;
+import componenttest.annotation.TestServlets;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.topology.database.container.DatabaseContainerType;
+import componenttest.topology.database.container.DatabaseContainerUtil;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.utils.PrivHelper;
+
+@RunWith(FATRunner.class)
+@Mode(TestMode.LITE)
+public class Packaging_Web extends JPAFATServletClient {
+    @Rule
+    public org.junit.rules.TestRule skipDBRule = new org.junit.rules.Verifier() {
```

Comment:
Maybe migrate this to `com.ibm.ws.jpa.tests.spec10.packaging_fat.common/fat/src/com/ibm/ws/jpa/tests/spec10/packaging/tests/SkipDatabaseRule.java`?

Reply:
Will just use the inline and get rid of the copy of SkipDatabaseRule.